<launch>

  <arg name="info_file" default="ur3e_robotiq_info.yaml"/>

  <!-- Launch interfacing nodes -->
  <node name="lik_ur_interface" pkg="ur3e_official_driver_relaxed_ik_demo" type="lik_ur_interface.py" output="screen">
    <param name="info_file_path" value="$(find lively_ik)/src/RelaxedIK/Config/info_files/$(arg info_file)"/>
    <param name="gripper_joint" value="gripper_finger1_joint"/>
    <param name="joint_step_delay" value="0.5" type="double"/>
  </node>

  <!-- Marker server -->
  <node name="marker_interface" pkg="ur3e_official_driver_relaxed_ik_demo" type="marker_interface.py" output="screen"/>

  <!-- Rviz -->
  <node type="rviz" name="rviz" pkg="rviz" args="-d $(find ur3e_official_driver_relaxed_ik_demo)/rviz/test.rviz" />

</launch>
